acm/acm-backend-lsp.el

```diff
@@ -230,8 +230,7 @@ If optional MARKER, return a marker instead"
 (defun acm-backend-lsp-snippet-expansion-fn ()
   "Compute a function to expand snippets.
 Doubles as an indicator of snippet support."
-  (and (boundp 'yas-minor-mode)
-       (symbol-value 'yas-minor-mode)
+  (and (bound-and-true-p 'yas-minor-mode)
```

Comment:
Change to 

```elisp
(and (boundp 'yas-minor-mode) 
         (symbol-value 'yas-minor-mode)
         yas-minor-mode))
```

Otherwise Github CI will failed, I guess Emacs in CI haven't define function `bound-and-true-p`